src/commands/dao_cmds/token.js

```diff
@@ -0,0 +1,151 @@
+const TaskList = require('listr')
+const { ensureWeb3 } = require('../../helpers/web3-fallback')
+const { getContract } = require('../../util')
+const listrOpts = require('../../helpers/listr-options')
+const chalk = require('chalk')
+
+const ZERO_ADDR = '0x0000000000000000000000000000000000000000'
+
+exports.command =
+  'token <token-name> <symbol> [decimal-units] [transfer-enabled]'
+
+exports.describe = 'Create a new minime token'
+
+exports.builder = yargs => {
+  return yargs
+    .positional('token-name', {
+      description: 'Full name of the new Token',
+    })
+    .positional('symbol', {
+      description: 'Symbol of the new Token',
+    })
+    .option('transfer-enabled', {
+      description: 'Whether the new token will have transfers enabled',
+      default: true,
+    })
+    .option('decimal-units', {
+      description: 'Total decimal units the new token will use',
+      default: 18,
+    })
+}
+
+exports.task = async ({
+  web3,
+  reporter,
+  tokenName,
+  symbol,
+  transferEnabled,
+  decimalUnits,
+  silent,
+  debug,
+}) => {
+  // Decode sender
+  const accounts = await web3.eth.getAccounts()
+  const from = accounts[0]
+
+  return new TaskList(
+    [
+      {
+        title: 'Deploy the MiniMeTokenFactory contract',
+        task: async (ctx, task) => {
+          let artifact = getContract('@aragon/os', 'MiniMeTokenFactory')
+          let contract = new web3.eth.Contract(artifact.abi)
+
+          const deployTx = contract.deploy({ data: artifact.bytecode })
+          const gas = await deployTx.estimateGas()
+
+          const deployPromise = deployTx.send({ from, gas })
+          deployPromise
+            .on('receipt', function(receipt) {
+              ctx.factoryAddress = receipt.contractAddress
+            })
+            .on('transactionHash', transactionHash => {
+              ctx.factoryTxHash = transactionHash
+            })
+            .on('error', function(error) {
+              throw error
+            })
+
+          task.output = `Waiting for the transaction to be mined...`
+          return deployPromise
+        },
+      },
+      {
+        title: 'Deploy the MiniMeToken contract',
+        task: async (ctx, task) => {
+          let artifact = getContract('@aragon/os', 'MiniMeToken')
+          let contract = new web3.eth.Contract(artifact.abi)
+
+          const deployTx = contract.deploy({
+            data: artifact.bytecode,
+            arguments: [
+              ctx.factoryAddress,
+              ZERO_ADDR,
+              0,
+              tokenName,
+              decimalUnits,
+              symbol,
+              transferEnabled,
+            ],
+          })
+          const gas = await deployTx.estimateGas()
+
+          const deployPromise = deployTx.send({ from, gas: gas + 1e6 })
```

Comment:
I'm not sure about `gas + 1e6`, will this work on mainnet, is it ok?

I was getting `The contract code couldn't be stored, please check your gas amount.`

Note that the estimated gas value was:
- on ganache, `3365974` for the factory and `3123814` for the token
- on rinkeby, `3365974` for the factory and `3089614` for the token